README.md

```diff
@@ -85,12 +80,11 @@ To enable Xdebug, set `PHP_ENABLE_XDEBUG=1` in .env file
 Xdebug is configured to call ip `xdebug.remote_host` on `9005` port (not use standard port to avoid conflicts),
 so you have to configure your IDE to receive connections from that ip.
 
-If you are using macOS, you can fill `xdebug.remote_host` with `host.docker.internal`, due to a network limitation on mac (https://docs.docker.com/docker-for-mac/networking/#port-mapping)
+If you are using macOS, you can fill `xdebug.remote_host` with `host.docker.internal`, due to a network limitation on mac (<https://docs.docker.com/docker-for-mac/networking/#port-mapping>)
```

Comment:
`<>` not need

Reply:
I'm following Markdown rules. Do I make your change or follow the rules?
[markdown md034.md](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md034.md)

Reply:
GitHub automatically add links. But it's OK, you can keep angle brackets 👍